osu.Game/Rulesets/Edit/DrawableEditRulesetWrapper.cs

```diff
@@ -69,7 +80,7 @@ private void removeHitObject(HitObject hitObject)
             drawableRuleset.Playfield.Remove(drawableObject);
             drawableRuleset.Playfield.PostProcess();
 
-            drawableRuleset.RegenerateAutoplay();
+            updateReplay();
```

Comment:
Is this still required? Because regeneration is now bound to the change handler.

Reply:
Likely not.